KarateTraining/src/test/java/abmarticle/abmarticle.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Testing the react herokuapp for creation , modification and removal
+
+  Background:
+    * url baseUrl
+    * def token = authInfo.accessToken
+
+    ##* callonce read('classpath:src/test/java/signintask/signin.feature@postSignInUser')
+    ##* def token = response.user.token
+
+  @CreateNewArticle
+  Scenario: Create a new article
+
+    * def article = read('classpath:abmarticle/abmNewArticleRequest.json')
+    ##* def signIn = call read('classpath:src/test/java/signintask/signin.feature@postSignInUser')
+
+    Given path 'articles'
+    And request article
+    And header Authorization = 'Bearer ' + token
```

Comment:
add "Bearer" only once in the karate-config in order to don't repeate this en every scenario